app/frontend/app/controllers/anime/quotes.js

```diff
@@ -37,6 +30,7 @@ export default Ember.ArrayController.extend(HasCurrentUser, {
           });
 
       quote.save();
+      this.get('content').addObject(quote);
```

Comment:
No, creating and saving the record will push it to the store. While this might update every property dependent on that via `model.@each`, it does not update the route model unless you actively reload it. - As far as I'm aware of.